src/data-loaders/RelationalDataLoader.js

```diff
@@ -94,12 +93,35 @@ const convertPostgresSQLResponseToColumnMetaData = (rows) => {
       label: row.name,
       name: row.name,
       type: row.dataTypeID,
-      typeName,
+      typeName: typeName ? typeName.toUpperCase() : "UNKOWN",
     };
   });
 };
-const executeSqlStatements = async (client, req) =>
-  Promise.mapSeries(req.statements, (statement) => client.query(statement));
+
+const injectVariables = (statement, variableMap) => {
+  const result = Object.keys(variableMap).reduce((statmnt, key) => {
+    if(variableMap[key] === null || variableMap[key] === false || variableMap[key] === true) {
+      return statmnt.replace(key, `${variableMap[key]}`)  
+    }
+    // @TODO: Differentiate number from string inputs...
+    return statmnt.replace(key, `'${variableMap[key]}'`)
```

Comment:
Do we need a `:` in front of `keys here?

Reply:
no we don't 😁 - keys already contain them